bin/wait_for_socket_file.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
Append dapr license

https://github.com/dapr/dapr/blob/master/tools/proto/generate.sh#L1

(that's for me too I always forget)